Description: Make fenix-fxi exit with a non-zero status on errors.
Forwarded: no
Author: Miriam Ruiz <little_miry@yahoo.es>
Last-Update: 2007-07-31

Index: fenix-0.92a.dfsg1/fxi/src/main.c
===================================================================
--- fenix-0.92a.dfsg1.orig/fxi/src/main.c	2007-08-28 14:40:57.000000000 +0200
+++ fenix-0.92a.dfsg1/fxi/src/main.c	2007-08-28 14:45:34.000000000 +0200
@@ -304,7 +304,7 @@
 				"agreement (GNU GPL version 2 or later).\n"
 				"See COPYING for license details.\n"),
 				argv[0]) ;
-			return 0 ;
+			return -1 ;
 		}
 	}
 
@@ -342,7 +342,7 @@
 				if (!dcb_load(dcbname))
 				{
 					gr_error (_("'%s': file does not exist or it's not a DCB version %d file or compatible"), filename, DCB_VERSION >> 8) ;
-					return -1 ;
+					return -2 ;
 				}
 			}
 		}
